float32 yaw_set
float32 pitch_set
float32 roll_set
float32 distance_set
float32 depth_set
bool final_command
